pkg/agent/util/net_windows.go

```diff
@@ -579,3 +582,61 @@ func parseRoutes(routesStr string) ([]Route, error) {
 	}
 	return routes, nil
 }
+
+func GetTransportIPNetDeviceByName(ifaceName string, ovsBridgeName string) (*net.IPNet, *net.Interface, error) {
```

Comment:
I feel better to move this function to `agent_XX.go` as it's not really generic, especially `GetVirtualInterfaceByName`. And why don't it reuse code in `GetIPNetDeviceByName`? It can be simplified as:
```
func getTransportIPNetDeviceByName(ifaceName string, ovsBridgeName string) (*net.IPNet, *net.Interface, error) {
	for _, name := range []string{ifaceName, fmt.Sprintf("vEthernet (%s)", ifaceName), ovsBridgeName} {
		ipNet, link, err := util.GetIPNetDeviceByName(ifaceName)
		if err == nil {
			return ipNet, link, nil
		}
		if !strings.Contains(err.Error(), "no such network interface") {
			return nil, nil, err
		}
	}
	return nil, nil, fmt.Errorf("unable to find local IP and device")
}
```